lib/rules/max-line-length/index.js

```diff
@@ -34,7 +34,11 @@ const rule = function (maxLength, options) {
 
     // Collapse all urls into something nice and short,
     // so they do not throw the game
-    const rootString = root.toString().replace(/url\(.*\)/ig, "url()")
+    const rootString = root.toString()
+    // Replace `url(something)` with `url()`
+    .replace(/url\(.*\)/ig, "url()")
+    // Replace `@import "something"` with `@import ""`
+    .replace(/\@import\s+['"].*['"]/ig, "@import ")
```

Comment:
Unfortunately, I do not think this regexp is exact enough. for example, `@import 'svg-something<id="horse">';` will become `@important ">';`, right?